package.json

```diff
@@ -85,14 +87,15 @@
     "markdown-it-container": "^3.0.0"
   },
   "devDependencies": {
-    "@types/markdown-it": "^12.2.3",
     "@babel/core": "^7.15.0",
+    "@types/markdown-it": "^12.2.3",
     "@types/vscode": "^1.72.0",
     "@types/vscode-notebook-renderer": "^1.72.0",
     "babel-loader": "^8.2.2",
-    "css-loader": "^2.1.1",
-    "mermaid": "^9.3.0",
+    "css-loader": "^6.7.3",
+    "mermaid": "^10.0.0",
     "mini-css-extract-plugin": "^2.2.2",
+    "npm-run-all": "^4.1.5",
```

Comment:
Please only include necessary changes in PRs to make them easier to review. The run-all change looks ok but belongs in a separate PR